apps/system/utils/netcmd.c

```diff
@@ -89,14 +89,6 @@ extern int iotivity_cloud_main(int argc, char *argv[]);
 #ifdef CONFIG_SYSTEM_NETDB
 extern int netdb_main(int argc, char *argv[]);
 #endif
-#ifdef CONFIG_EXTERNAL_WAKAAMA_EXAMPLE
-#ifdef CONFIG_LWM2M_CLIENT_MODE
-extern int lwm2m_client_main(int argc, char *argv[]);
-#endif
-#ifdef CONFIG_LWM2M_SERVER_MODE
-extern int lwm2m_server_main(int argc, char *argv[]);
-#endif
-#endif
```

Comment:
It would be good to leave the legacy application in place. That way, developers can choose whether to use it with DM framework, or develop their own application with wakaama core.